codex/raw/marketplace.md

```diff
@@ -0,0 +1,269 @@
+---
+title: CODEX-MARKETPLACE
+name: Codex Storage Marketplace
+status: raw
+tags: codex
+editor: Dmitriy <dryajov@status.im>
+contributors:
+- Mark <mark@codex.storage>
+- Adam <adam.u@status.im>
+- Eric <ericmastro@status.im>
+- Jimmy Debe <jimmy@status.im>
+---
+
+## Abstract
+
+This specification describes a method for Codex storage providers and client nodes to participate in a storage marketplace. 
+The goal is to create a storage marketplace that promotes durability.
+
+## Motivation
+The Codex network aims to create a peer-to-peer storage engine with strong data durability, 
+data persistence guarantees and node storage incentives.
+To reach this goal, a data availability and retrieval mechanism is needed.
+Support for light clients, like mobile devices should also be embraced.
+The protocol should remove complexity to allow for simple implementation and 
```

Comment:
I would remove this phrasing. This isn't a concrete goal that can be properly quantified.